.github/workflows/ci.yml

```diff
@@ -191,7 +190,7 @@ jobs:
 
   build_test_all_windows:
     needs: setup
-    if: needs.setup.outputs.should-run == 'true' && needs.setup.outputs.ci-stage == 'postsubmit'
+    if: needs.setup.outputs.should-run == 'true' && contains(['push', 'workflow_dispatch'], github.event_name)
```

Comment:
I assume "contains" here returns true if _either_ string is found? (Having some shorthand for "not presubmit" or "postsubmit" would still be helpful)

Reply:
The first argument is the list to search and the second the thing to search for. The argument order trips me up every time.

https://docs.github.com/en/actions/learn-github-actions/expressions#contains